.bithoundrc

```diff
@@ -0,0 +1,13 @@
+{
+  "critics": {
+    "lint": {
+      "engine": "eslint"
+    },
+    "wc": {
+      "limit": 30000
+    }
+  },
+  "ignore": [
+    "test/**"
```

Comment:
bitHound's default config marks files in `**/test/**` as "test" files and scores them accordingly, which is okay in my opinion, there is no need to ignore them.